pyaedt/generic/general_methods.py

```diff
@@ -1574,6 +1574,48 @@ def _check_installed_version(install_path, long_version):
     return False
 
 
+def install_with_pip(package_name, package_path=None, upgrade=False, uninstall=False):  # pragma: no cover
+    """Install a new package using pip.
+    Useful to install a package from AEDT Console without launching the python environment.
+
+    Parameters
+    ----------
+    package_name : str
+        Name of the package to install.
+    package_path : str, optional
+        Package path for GitHub download. Example ``git+https://......``.
+    upgrade : bool, optional
+        Whether to upgrade the package or not.
+    uninstall : bool, optional
+        Whether to install the package or uninstall the package.
```

Comment:
```suggestion
        Whether to install the package or uninstall the package.
```
This is confusing. The default is ``False``, but what does this do if you are choosing between installing or unintalling?

Reply:
2 options: install a package or uninstall a package. True, uninstall, False install